Project/Source Files/Seach/Search.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Search.swift
+//  NetguruCollegeApp
+//
+
+import Foundation
+
+protocol SearchDelegate: AnyObject {
+    func repoFound(data: Data?)
+    func userFound(data: Data?)
+}
+
+class Search {
+    private let apiClient = APIClient()
+    func makeSearch(nameContains name: String, type: SearchRequestType) {
+        if type == .projectName {
+            seachProject(nameContains: name)
+        } else if type == .userName {
+            searchUser(nameContains: name)
+        }
+    }
+    func seachProject(nameContains name: String) {
+        let request = SearchProjectRequest()
+        let convertedName = convertProjectName(name: name)
+        let parameters = ["q": convertedName,
+                          "order": "desc"]
+        request.parameters = parameters
+        apiClient.sendRequest(request: request,
+                              success: { data in
+            print("success")
+        }) { error in
+            if error != nil {
+                print(error!)
+            } else {
+                print("Search repo error. NIL in responce.")
+            }
+        }
+    }
+    func searchUser(nameContains name: String) {
```

Comment:
this function needs a way to return data, how about adding completion?
the same applies to search project